docs/deployment/containers/docker.txt

```diff
@@ -191,6 +191,13 @@ Here's how you would mount ``crate.yml``::
 Here, ``/srv/crate/config/crate.yml`` is an example path, and should be
 replaced with the path to your host machine ``crate.yml`` file.
 
+Healthcheck
+-----------
+
+The official `CrateDB Docker image`_ ships with a configured liveness
+`Docker healthcheck`_. The purpose of this check is to signal the situations
```

Comment:
`signal` -> `indicate`

Reply:
@nslater what do you think? I find `signal` to be expressive enough

Reply:
This health check will flag that CrateDB process might...

Reply:
also change it to "Docker health check"